app/controllers/application_controller.rb

```diff
@@ -1,5 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  
+  include ApplicationControllerImpl
+  # include SafeLoggerRaygun
```

Comment:
Don't introduce commented out code. Please eliminate this habit entirely, consider it an essential step for becoming a pro